samples/samples-outofproc/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# Azure Functions localsettings file
+local.settings.json
+
+# User-specific files
+*.suo
```

Comment:
Do you need this? Most of these are duplicated in the top level gitignore already. If there's ones that are relevant to just this folder then you can keep them, but delete the rest.